examples/tutorials/Part 11c - Secure Classification with Syft Keras and TFE: Private Prediction Client.ipynb

```diff
@@ -0,0 +1,206 @@
+{
```

Comment:
<p>Before querying the model, you have to connect to it. To do so, you first create a client. Then, you define the exact same three TFEWorkers that we defined in the last notebook (<code>alice</code>,&nbsp;<code>bob</code>, and&nbsp;<code>carol</code>). Finally call&nbsp;<code>connect_to_model</code>.</p><p><br></p><p>we also need more explanation about what "connect_to_model" is doing.</p>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/2170/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>